### ISSUE TITLE
YAMLException for ng_controller_name RegExp

### ISSUE THREAD

**Issue by JamieMason** [Gillespie59/eslint-plugin-angular#98]
If I have the following item in my .eslintrc and run `eslint src/app`

```json
"angular/ng_controller_name": [2, /[A-Z].*Controller$/]
```

The following error is reported by eslint

    /usr/local/lib/node_modules/eslint/lib/config.js:76
                    throw e;
                          ^
    YAMLException: Cannot read config file: /Users/jmason/Development/ssbt-bottom-screen/.eslintrc
    Error: JS-YAML: missed comma between flow collection entries at line 394, column 44:
         ... gular/ng_controller_name": [2, /[A-Z].*Controller$/],
                                             ^

If I remove that entry it works fine, could you let me know what I'm doing wrong please? Does the value need to be a String value which would passed to the `RegExp` constructor? for example.

```json
"angular/ng_controller_name": [2, "[A-Z].*Controller$"]
```

```javascript
new RegExp('[A-Z].*Controller$');
```

Thanks a lot.

**Comment by Gillespie59** [comment]
Hi @JamieMason 

Can you try by wrapping the regexp with single or double quotes ? 

```yaml
"angular/ng_controller_name": [2, "/[A-Z].*Controller$/"]
```

Manu

**Comment by Gillespie59** [comment]
Is this answer enough for you? 
Can I close this issue ?

**Comment by JamieMason** [comment]
Sorry for the delay @Gillespie59, tested with 0.0.8 and it is now working - thanks.
